ui/css/itcss/components/network.scss

```
@@ -138,15 +138,15 @@
   width: 100%;
   height: 1px;
   margin: 10px 0;
-  background-color: var(--scorpion);
+  background-color: var(--color-border-default);
 }
 
 .network-dropdown-title {
   @include H4;
 
   height: 25px;
   width: 120px;
-  color: var(--white);
+  color: var(--color-background-default);
```

I think you may want to use overlay.inverse here if it's text on top of the component that uses overlay.default or overlay.alternative

```suggestion
  color: var(--color-overlay-inverse);
```